lib/runtime/wazero/imports_test.go

```diff
@@ -369,6 +369,41 @@ func Test_ext_crypto_ecdsa_verify_version_2_Table(t *testing.T) {
 	}
 }
 
+func Test_ext_crypto_ecdsa_generate_version_1(t *testing.T) {
+	// TODO: fix this
+	t.Skip("host API tester does not yet contain rtm_ext_crypto_ecdsa_generate_version_1")
+
+	inst := NewTestInstance(t, runtime.HOST_API_TEST_RUNTIME)
+
+	idData := []byte(keystore.AccoName)
+	ks, _ := inst.Context.Keystore.GetKeystore(idData)
```

Comment:
Is this error deliberately ignored? Otherwise let's take the error and add require.NoError